server/src/main/java/org/opensearch/client/node/NodeClient.java

```diff
@@ -56,7 +60,9 @@
  * @opensearch.internal
  */
 public class NodeClient extends AbstractClient {
-
+    private static final String SPAN_NAME_PREFIX_TRANSPORT_ACTION = "transport_action_";
```

Comment:
Please extract all our custom tracing attributes to some kind of registry (please check `SemanticAttributes` as an example)